lxd/main_cluster.go

```diff
@@ -136,7 +137,16 @@ func (c *cmdClusterEdit) Run(cmd *cobra.Command, args []string) error {
 
 	var nodes []db.RaftNode
 	err = database.Transaction(func(tx *db.NodeTx) error {
-		var err error
+		config, err := node.ConfigLoad(tx)
+		if err != nil {
+			return err
+		}
+
+		clusterAddress := config.ClusterAddress()
+		if clusterAddress == "" {
+			return fmt.Errorf(`LXD is not clustered (missing "cluster.https_address") - set up clustering first with "lxc cluster enable <host>"`)
```

Comment:
We usually don't like recommending specific `lxc` commands on server-side errors. Also this may have the side effect of random users enabling clustering just so they can run the edit command.

I think I'd prefer the error be something along the lines of `fmt.Errorf("Can't edit cluster configuration as server isn't clustered")`

Reply:
Ah yes I'd not remembered that. I suggested adding the bit about the  cluster.https_address because this tool is used when things go wrong and thought it'd be good to be transparent about why we think its not clustered, in case that setting was removed too.

Reply:
Yeah, the `missing cluster.https_address` is probably fine to keep, it's the `lxc cluster enable` bit that's problematic.

Reply:
Yep agreed. Thanks

Reply:
I've changed it to `Can't edit cluster configuration as server isn't clustered (missing "cluster.https_address" config)`